src/Walker/Collector.C

```diff
@@ -29,69 +29,13 @@ CkReduction::reducerType PDFMerger;
 using walker::Collector;
 
 void
-Collector::chareOrd( const std::vector< tk::real >& ord )
+Collector::chareOrd( const std::vector< tk::real >& ord,
```

Comment:
Given by first line documenting the function:
_Chares contribute ordinary moments and ordinary PDFs_